Framework/API/test/AsynchronousTest.h

```diff
@@ -11,12 +11,21 @@
 #include <Poco/NObserver.h>
 #include <Poco/Thread.h>
 
+#include <condition_variable>
+#include <mutex>
+
 using namespace Mantid::Kernel;
 using namespace Mantid::API;
 using namespace std;
```

Comment:
Can remove `using namespace std`? We shouldn't encourage its use in headers, even in tests.